inc/auth.class.php

```diff
@@ -657,16 +657,20 @@ function login($login_name, $login_password, $noauto = false, $remember_me = fal
                            AuthLdap::IDENTIFIER_LOGIN => $ldap_method["login_field"],
                         ],
                      ];
-                     $user_dn
-                        = AuthLdap::searchUserDn($ds,
-                                                 ['basedn'      => $ldap_method["basedn"],
-                                                       'login_field' => $ldap_method['login_field'],
-                                                       'search_parameters'
-                                                                     => $params,
-                                                       'user_params'
-                                                         => ['method' => AuthLDAP::IDENTIFIER_LOGIN,
-                                                                  'value'  => $login_name],
-                                                         'condition'   => $ldap_method["condition"]]);
+                     try {
+                        $user_dn
+                           = AuthLdap::searchUserDn($ds,
+                                                   ['basedn'      => $ldap_method["basedn"],
+                                                         'login_field' => $ldap_method['login_field'],
+                                                         'search_parameters'
+                                                                        => $params,
+                                                         'user_params'
+                                                            => ['method' => AuthLDAP::IDENTIFIER_LOGIN,
+                                                                     'value'  => $login_name],
+                                                            'condition'   => $ldap_method["condition"]]);
+                     } catch (\RuntimeException $e) {
+                        $user_dn = false;
```

Comment:
I think nothing gets logged as is.

Reply:
We had this case with Flyve MDM. 

I need to do tests to find if we need to update our  code to provide an extra argument to our call ro tje api endpoint initSession.

But it is clear that something must catch the possible exception thrown by this call

Reply:
I do not understand. I'm just saying you should call a logError or something similar (see similar try/catch in authldap class).

Reply:
Done